husky_components/navigation/main.js

```diff
@@ -982,6 +992,8 @@ define(function() {
          * Remove data navigation
          */
         removeDataNavigation: function() {
+            if (!!this.dataNavigationIsLoading) return;
```

Comment:
Why is that necessary? Shouldn't it be possible to remove the data navigation when it is currently loading?